reflex/testing.py

```diff
@@ -398,7 +398,7 @@ def frontend(self, driver_clz: Optional[Type["WebDriver"]] = None) -> "WebDriver
             )
         if self.frontend_url is None:
             raise RuntimeError("Frontend is not running.")
-        driver = driver_clz() if driver_clz is not None else webdriver.Chrome()
+        driver = driver_clz() if driver_clz is not None else webdriver.Firefox()
```

Comment:
not sure if we want to commit this...

Reply:
Fixed this, just had it locally for testing